ada-project-docs/wave_05.md

```diff
@@ -10,6 +10,16 @@ They contain a title to name the goal.
 
 Our goal for this wave is to be able to create, read, update, and delete different goals.
 
+## Writing Tests
+
+This wave requires you to complete **four** tests. 
+- The tests you need to write are scaffolded in the `test_wave_05.py` file. 
+  - These tests are currently skipped with `@pytest.mark.skip(reason="test to be completed by student")` and the function body has `pass` in it. Once you implement these tests you should remove the `skip` decorator and the `pass`.
+- For the tests you write, use the requirements in this document to guide your test writing. 
+  - Pay attention to the exact shape of the expected JSON. Double-check nested data structures and the names of the keys for any mispellings.
+- You can model your tests off of the Wave 01 tests for Tasks.
+- Some tests use a fixture named `one_goal` that is defined in `tests/conftest.py`. This fixture saves a specific goal to the test database.
```

Comment:
Feel free to ignore, but maybe add a link to the docs for a fixture as students haven't had fixtures yet.

```suggestion
- Some tests use a [fixture](https://docs.pytest.org/en/6.2.x/fixture.html) named `one_goal` that is defined in `tests/conftest.py`. This fixture saves a specific goal to the test database.
```